src/ferp/util/pure.js

```diff
@@ -0,0 +1,23 @@
+import { memoizeStore, memoize, getMemoized } from './memoize.js';
```

Comment:
It might be helpful to have `pureGetStore(pureFn)` and `pureSetStore(store, pureFn)`. I don't like exporting that symbol key, and would rather keep that portion private.

Reply:
I've changed my mind, pureSetStore shouldn't be a thing.